src/modules/kvs/kvs.c

```diff
@@ -1239,6 +1245,60 @@ static void heartbeat_sync_cb (flux_future_t *f, void *arg)
     flux_future_reset (f);
 }
 
+static void checkpoint_period_cb (flux_reactor_t *r,
```

Comment:
Any reason this stuff was not moved to a new source file?   It seems like it could be fairly self-contained and kvs.c is already really long.

Reply:
it seemed just below the threshold for a new file, but it's close.  I'll splice out into a new file.